docs/userguide/canvas.rst

```diff
@@ -1232,7 +1232,7 @@ the external monitoring system.
 
     class MonitoringIdStampingVisitor(StampingVisitor):
         def on_signature(self, sig, **headers) -> dict:
-            return {'monitoring_id': uuid4(), 'stamped_headers': ['monitoring_id']}
+            return {'monitoring_id': uuid4().hex, 'stamped_headers': ['monitoring_id']}
```

Comment:
In 5.3, UUIDs should be serializable using the JSON serializer. See https://github.com/celery/kombu/pull/1575.
You should install Kombu from master to verify.